src/store/index.js

```diff
@@ -22,4 +22,18 @@ export function getMessageSubject(message: Message): ?string {
   return subjectHeader && subjectHeader.value;
 }
 
+export function getMessageSender(message: Message): ?string {
+  const { headers } = message.payload;
+  const senderHeader = headers.find(header => header.name === 'From');
+  return senderHeader && senderHeader.value;
+}
+
+export function formatSender(sender: String){
```

Comment:
Flow uses lowercase `string`, for one. But I think you’re passing in a maybe-string (`?string`) too.